Dockerfile

```diff
@@ -4,4 +4,4 @@ COPY package*.json ./
 RUN npm ci
 COPY . .
 
-ENTRYPOINT ["node", "/index.js"]
+ENTRYPOINT ["node", "/dist/index.js"]
```

Comment:
**issue (blocking):** dont forget to build the javascript code from the typescript one, we should add before this entrypoint: `RUN npm run build`

Reply:
Added